### ISSUE TITLE
Issue with _req & ktlCond on form input

### ISSUE THREAD

**Issue by CSWinnall** [cortexrd/Knack-Toolkit-Library#308]
Hi Norm 
Just had a call with Carl and we are trying to get _req to work with ktlCond on a form input: _req=[field_3165], [ktlCond, is, Required, field_3135, view_5060] 

Where field 3135,is in the same view as field I want to make required, is a multi choice field with values Received & Required.  I can't get it to work. It works fine without the ktlCond. 

Craig

**Comment by chprojectsupport** [comment]
Yes, sorry, I broke it again 🤩Carl HolmesOn 28 Jun 2024, at 20:57, Craig Winnall ***@***.***> wrote:﻿
Hi Norm
Just had a call with Carl and we are trying to get _req to work with ktlCond on a form input: _req=[field_3165], [ktlCond, is, Required, field_3135, view_5060]
Where field 3135,is in the same view as field I want to make required, is a multi choice field with values Received & Required.  I can't get it to work. It works fine without the ktlCond.
Craig
